docker-compose/docker-compose.yml

```diff
@@ -76,12 +78,19 @@ services:
             - 8099:8099
 
     share:
-        image: alfresco/alfresco-share:6.0
+        image: alfresco-share-test
         mem_limit: 1g
         environment:
-            - REPO_HOST=alfresco
-            - REPO_PORT=8080
-            - "CATALINA_OPTS= -Xms500m -Xmx500m"
+            REPO_HOST: "alfresco"
+            REPO_PORT: "8080"
+            JAVA_OPTS: "
+                -Xms500m
+                -Xmx500m
+                -Dalfresco.host=localhost
+                -Dalfresco.port=8082
+                -Dalfresco.context=alfresco
+                -Dalfresco.protocol=http
```

Comment:
Is this correct? Would it normally be https externally?

Reply:
We are currently not using HTTPS for our docker-compose deployment.